libs/model/src/models/reaction.ts

```diff
@@ -172,17 +172,18 @@ export default (
       updatedAt: 'updated_at',
       indexes: [
         { fields: ['address_id'] },
-        {
-          fields: [
-            'community_id',
-            'address_id',
-            'thread_id',
-            'proposal_id',
-            'comment_id',
-            'reaction',
-          ],
-          unique: true,
-        },
+        // to avoid relation "reactions_community_id_address_id_thread_id_proposal_id_comment" already exists
```

Comment:
If you add an explicit name i.e. (same name as in prod) `name: 'reactions_unique',` this error won't be thrown anymore.